tests/testthat/test-knitr_extended_formats.R

```diff
@@ -0,0 +1,32 @@
+regexes <- list(
```

Comment:
this is overly complicated... please just use the text in the test directly, i.e. make these tests more DAMP:

https://stackoverflow.com/q/6453235/3576984

Reply:
I was just trying to be internally consistent. This is what we do in `test-knitr_formats.R`, so thought it is expected that `test-knitr_extended_formats.R` be similar.

Will change that.

Reply:
even in the other file I'm not sure it's the best idea... but even less so in this new file. thanks.